src/components/BracketCard/BracketCard.styles.ts

```diff
@@ -1,39 +1,39 @@
 import { cnb } from 'cnbuilder';
 
-export const root = 'su-relative su-mx-auto';
+export const root = 'su-group su-relative su-mx-auto';
 export const grid = (textOnLeft: boolean) => cnb(
   'su-flex su--mt-[40vw] sm:su-mt-0 sm:su-ml-0 su-w-[calc(100%_+_7rem)] sm:su-w-full sm:su-grid sm:su-grid-cols-12 sm:su-absolute sm:su-top-0 sm:su-bottom-[4vw] 3xl:su-bottom-70',
   textOnLeft ? 'su--mr-70' : 'su--ml-70',
 );
 
 export const bracket = (textOnLeft: boolean) => cnb(
-  'su-relative su-z-10 su-w-[13rem] su-shrink-0 sm:su-col-span-2 sm:su-w-[120%]',
+  'su-z-10 su-w-[13rem] su-shrink-0 sm:su-col-span-2 sm:su-w-[120%]',
   textOnLeft ? 'su-order-2 sm:su-col-end-12 sm:su-justify-self-end' : 'sm:su-col-start-2',
 );
 
 export const contentCard = (textOnLeft: boolean) => cnb(
-  'su-relative su-z-10 sm:su-col-span-9 su-border-b-2 su-border-white',
+  'su-z-10 sm:su-col-span-9 su-border-b-2 su-border-white',
   textOnLeft ? 'su-order-1 sm:su-col-start-1' : 'sm:su-col-start-4',
 );
 
 export const imageGrid = (textOnLeft: boolean) => cnb(
   'sm:su-ml-0 sm:su-pt-[6vw] 3xl:su-pt-100',
-  textOnLeft ? 'su--mr-20' : 'su--ml-20',
+  textOnLeft ? 'su--mr-20 sm:su-mr-0' : 'su--ml-20 sm:su-ml-0',
```

Comment:
Previously I forgot to reset the negative margins on the image back to 0 from SM breakpoint and up. We added the negative margins on XS to have the image flushed to the edge of the browser.